modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -60,37 +59,38 @@ Only segment level customization is supported.
 [[target-at-host]]
 == Target at Host
 
-These are application message structure (JSON or XML) to which inbound EDI transactions are translated into and sent to your back-end applications.
+The application message structure (JSON or XML) to which inbound EDI transactions are translated and sent to your back-end applications.
 
 [%header%autowidth.spread]
 |===
 |Setting |Description |Default | Required
-|Format a|Select a format from the drop-down list:
+|Format a|The selected application message structure format:
 
-* *XML* 
+* *XML*
 * *JSON* |None |Yes
-|Schema |Import the XSD or JSON schema file for the message type. |None |Yes
-|Name |Partner Manager automatically generates a message type name from the schema file name. You can override it if you need to. |None |Yes
+|Schema |The imported XSD or JSON schema file for the message type. |None |Yes
+|Name |Partner Manager automatically generates a message type name from the schema file name. This value can be modified. |None |Yes
 |===
 
 [[source-at-host]]
-== Source at Host 
+== Source at Host
+Application message structures (JSON or XML) from your back-end applications that get translated into  outbound EDI transactions and sent to your trading partners.
```

Comment:
ditto re. complete sentence and intro.
omit extra space as well

Reply:
Done